lib/Dialect/Calyx/Export/CalyxEmitter.cpp

```diff
@@ -545,8 +541,12 @@ void Emitter::emitComponent(ComponentOp op) {
                 SubLibOp, ShruLibOp, RshLibOp, SrshLibOp, LshLibOp, AndLibOp,
                 NotLibOp, OrLibOp, XorLibOp, WireLibOp>(
               [&](auto op) { emitLibraryPrimTypedByFirstInputPort(op); })
-          .Case<MultPipeLibOp, DivUPipeLibOp, DivSPipeLibOp>(
+          .Case<MultPipeLibOp>(
               [&](auto op) { emitLibraryPrimTypedByFirstOutputPort(op); })
+          .Case<RemUPipeLibOp, RemSPipeLibOp, DivUPipeLibOp, DivSPipeLibOp>(
```

Comment:
Signed remainder/divpipe should be `std_sdiv_pipe`

Ref: https://github.com/cucapra/calyx/blob/574e4a901a6ac5b0fa8adc115471267b17a0c3e8/primitives/binary_operators.futil#L128